src/base/utils/foreignapps.cpp

```diff
@@ -108,18 +113,17 @@ namespace
         if (res == ERROR_SUCCESS)
         {
             ++cMaxSubKeyLen; // For null character
-            LPWSTR lpName = new WCHAR[cMaxSubKeyLen];
+            std::wstring lpName(cMaxSubKeyLen + 5, 0);
             DWORD cName;
 
             for (DWORD i = 0; i < cSubKeys; ++i)
             {
                 cName = cMaxSubKeyLen;
-                res = ::RegEnumKeyExW(handle, i, lpName, &cName, NULL, NULL, NULL, NULL);
+                res = ::RegEnumKeyExW(handle, i, lpName.data(), &cName, NULL, NULL, NULL, NULL);
                 if (res == ERROR_SUCCESS)
-                    keys.push_back(QString::fromWCharArray(lpName));
+                    keys.push_back(QString::fromWCharArray(lpName.c_str()));
```

Comment:
FYI, there is `QString QString::fromStdWString(const std::wstring &str)` method.